misc/pytorch_toolkit/text_recognition/text_recognition/models/model.py

```diff
@@ -20,6 +20,7 @@
 import torch.nn as nn
 
 from .backbones.resnet import (CustomResNetLikeBackbone, ResNetLikeBackbone)
+from .backbones.cnn import HETRBackbone
```

Comment:
Please, change filename from `cnn` to `hetr` also, and rename class name in the backbones mapping and in the config (from `cnn` to `hetr`)